src/renderer/components/menu/menu-actions.tsx

```diff
@@ -124,30 +125,32 @@ export class MenuActions extends React.Component<MenuActionsProps> {
     return (
       <>
         {this.renderTriggerIcon()}
-        <Menu
-          htmlFor={this.id}
-          isOpen={this.isOpen} open={this.toggle} close={this.toggle}
-          className={menuClassName}
-          usePortal={autoClose}
-          closeOnScroll={autoClose}
-          closeOnClickItem={autoCloseOnSelect ?? autoClose }
-          closeOnClickOutside={autoClose}
-          {...menuProps}
-        >
-          {children}
-          {updateAction && (
-            <MenuItem onClick={updateAction}>
-              <Icon material="edit" interactive={toolbar} tooltip="Edit"/>
-              <span className="title">Edit</span>
-            </MenuItem>
-          )}
-          {removeAction && (
-            <MenuItem onClick={this.remove}>
-              <Icon material="delete" interactive={toolbar} tooltip="Delete"/>
-              <span className="title">Remove</span>
-            </MenuItem>
-          )}
-        </Menu>
+        <RenderDelay>
+          <Menu
```

Comment:
While menu trigger icon is rendered and visible instantly, menu itself which appears only on user click may be postponed to render.

Reply:
Decided to use `Menu` with `RenderDelay` because it triggers a lot of actions such as bunch of `addEventListener` for each menu instance.